### PR TITLE
cf-editor-form-field-dropdown-sample

### PR DESCRIPTION
**Pull request by VladimirZaets**
[Workday] Support values coming from custom servlet (ACS AEM Commons) to display in the Enum data type in the new CF Editor

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/ExtensionRegistration.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  console.log("await guestConnection?.host init")
+
+  const init = async () => {
+    const guestConnection = await register({
+      id: extensionId,
+      methods: {
+        field: {
+          getDefinitions: () => ([
+            {
+              fieldNameExp: '^label01',
```
Comment (lenaorobei) [inline cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/ExtensionRegistration.js:20]: Shouldn't it be more generic?

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/utils.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+* <license header>
```
Comment (lenaorobei) [inline cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/utils.js:2]: Please fix license header.

File: cf-editor-form-field-dropdown-sample/README.js
Comment (lenaorobei) [inline cf-editor-form-field-dropdown-sample/README.js:1]: Is this needed?

File: cf-editor-form-field-dropdown-sample/extension-manifest.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "aem-add-btn-ext",
```
Comment (lenaorobei) [inline cf-editor-form-field-dropdown-sample/extension-manifest.json:2]: Please fix the name and description.

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/Dropdown.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      <div className={'dropdown-field-wrapper'}>
+        <ComboBox
+          label="Favorite Animal"
+          maxWidth="100%"
+          minWidth={"auto"}
+          UNSAFE_className={"dropdown-field"}
+          defaultItems={items}
+          onOpenChange={(isOpen, menuTrigger) => {
+              if (isOpen) {
+                connection.host.field.setHeight(defaultHeight + (6 * 20))
```
[the remark this answers is not in the document]
Comment (VladimirZaets) [reply cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/Dropdown.js:45]: @fe-lix- yes, just forgot to update it, thanks

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/index.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  const mockIms = {}
+
+  // render the actual react application and pass along the runtime object to make it available to the App
+  ReactDOM.render(
+    <App runtime={mockRuntime} ims={mockIms} />,
+    document.getElementById('root')
+  )
+}
+
+function bootstrapInExcShell () {
```
[the remark this answers is not in the document]
Comment (VladimirZaets) [reply cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/index.js:39]: @lenaorobei it is used in initialization

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/ExtensionRegistration.js (hunk 1 above)
Comment (VladimirZaets) [reply cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/components/ExtensionRegistration.js]: done

File: cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (lenaorobei) [reply cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/index.js:46]: It's not used, we always call `bootstrapRaw`. In fact, the code of this file can be as simple as
```
import 'core-js/stable'

window.React = require('react')
import ReactDOM from 'react-dom'

import App from './components/App'
import './index.css'

ReactDOM.render(
  <App />,
  document.getElementById('root')
)
```

[the remark this answers is not in the document]
Comment (lenaorobei) [reply cf-editor-form-field-dropdown-sample/src/aem-cf-editor-1/web-src/src/index.js:46]: Looks like we need to look into our templates and check the code that's being generated.

**Review by lenaorobei — APPROVED** [review]

**Pull request merged**